ruuvi_examples/ruuvi_firmware/main.c

```diff
@@ -1,18 +1,5 @@
-/* Copyright (c) 2015 Nordic Semiconductor. All Rights Reserved.

```

Comment:
Nordic Semiconductor license generally forbids removing any copyright notices from the files. RuuviTag FW is modified from Nordic Semi example, therefore I'd prefer to leave the header.

Reply:
That's fine with me, it just seems to be a long way until you get to some meaningful code.
1) How much of original code from Nordic Semiconductor remains? 
2) could it be include via an #include ?
3) could it be pasted at the end?
whatever.

Reply:
Most editors/IDE's automatically collapse recognized copyright/license comment blocks, leaving the first line visible, so I don't think it's a big issue to leave it as-is there

Reply:
Some  editors/IDE's automatically...

4. should the be a copyright notice re ruuvi innovations? 
5. data is 2015 should that be revised?

Reply:
1. Probably very little, however I'm not an expert in international copyright laws so I have no idea if there is a point at which derived code ceases to be a derived code and becomes original product. Let's keep the attribution as it was and if necessary add another attribution to modifications.

2. Maybe, or even a link could do. Some of the Nordic license headers have very explicit "this heading must not be removed", let's honor the spirit of those headers.

3. Maybe, but it's a convention to keep any copyright information at the top of a file.

4. It could be attached after Nordic Semi, maybe "copyright Ruuvi & Contributors, see ... " and link to the licenses/contributors. 

5. If a new copyright notice is made maybe yes. 

In general I'd consider https://github.com/ruuvi/ruuvitag_fw/blob/master/licenses.md to be authoritative list of contributors and licensing, any additional copyright notice could be link to that.

Reply:
Copyright notice for Nordic Semi placed near the beginning of the source (after version comment)